pkg/vsphere/performance/host_metrics.go

```diff
@@ -66,18 +68,18 @@ type HostMetricsInfo struct {
 // HostMetricsProvider returns CPU and memory metrics for all ESXi hosts in a cluster
 // via implementation of the MetricsProvider interface.
 type HostMetricsProvider struct {
-	session *session.Session
+	*session.Session
```

Comment:
Doesn't look as though session is being used for anything beyond getting the `vim25.Client` so we could simply require and store the vim25 client to reduce cross package dependencies.

Reply:
👍